extensions/gmsa-dc/v1/Promote-DC.ps1

```diff
@@ -0,0 +1,51 @@
+###################################################################################
+# This script will automate the creation of a Forest, Domain, and Domain Controller
+#
+# This is only for automated e2e testing.  DO NOT use this for production.
+# Jeremy Wood (JeremyWx)
+# Version: 1.0.0.0
+###################################################################################
+
+# Create a working directory and change to it
+mkdir C:\gmsa
+Set-Location -Path C:\gmsa
+
+# Uncomment line below for troubleshooting
+Start-Transcript -Path "C:\gmsa\Promote.txt"
+
+# Download gMSA Setup Script
+Invoke-WebRequest -UseBasicParsing https://raw.githubusercontent.com/JeremyWx/aks-engine/master/extensions/gmsa-dc/v1/Setup-gMSA.ps1 -OutFile Setup-gMSA.ps1
+
+#Install NuGet
+Install-PackageProvider -Name NuGet -Force
+
+#Create Local Administrator
+$admpassword = ( "K8s" + -join ((48..57) + (97..122) | Get-Random -Count 64 | ForEach-Object {[char]$_}) )
+$admpassword_secure = ( $admpassword | ConvertTo-SecureString -AsPlainText -Force)
+New-LocalUser -Name gmsa-admin -Password $admpassword_secure
+Add-LocalGroupMember -Group "Administrators" -Member gmsa-admin
+Start-Sleep -Seconds 5

```

Comment:
Why is this needed?

Reply:
Honestly, I don't remember why that was added.  It has been removed.